guides/common/modules/proc_estimating-the-size-of-a-backup.adoc

```diff
@@ -12,6 +12,13 @@ A full backup requires space to store the following data:
 * Compressed {Project} database and configuration files
 * An extra 20% of the total estimated space to ensure a reliable backup
 
+ifdef::satellite[]
+[NOTE]
+----
+Backup does not include container images for {insights-iop}.
+----
```

Comment:
```suggestion
====
Backups do not include container images for {insights-iop}.
====
```

Refs https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/#admonition-syntax & I made a minor suggestion to the wording.